ArduPlane/Parameters.cpp

```diff
@@ -1274,6 +1274,15 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(precland, "PLND_", 35, ParametersG2, AC_PrecLand),
 #endif
 
+#if AP_RANGEFINDER_ENABLED
+    // @Param: RNGFND_LND_ORNT
+    // @DisplayName: rangefinder landing orientation
+    // @Description: The orientation of rangefinder to use for landing detection. Should be set to rangefinder_orientation() for normal downward facing rangefinder and ROTATION_PITCH_180 for rearward facing rangefinder for quadplane tailsitters. Custom orientation can be used with CUST_ROT_ENABLE. The orientation must match at least one of the available rangefinders.
```

Comment:
I think the description will need to use the same vocabulary as the @Values tag, e.g.:
```suggestion
    // @Description: The orientation of rangefinder to use for landing detection. Should be set to Down for normal downward facing rangefinder and Back for rearward facing rangefinder for quadplane tailsitters. Custom orientation can be used with Custom1 or Custom2. The orientation must match at least one of the available rangefinders.
```

Reply:
applied, thanks!